### PR TITLE
Exclude HTML element for css selector

### PR DESCRIPTION
**Pull request by nkashyap**
Following code exclude `<html>` tag from css selector.

### CODE REVIEW

File: src/isElement.js [hunk 1]
```diff
@@ -5,13 +5,17 @@ export function isElement( el )
 {
   let isElem;
 
-  if ( typeof HTMLElement === 'object' )
+  if ( el && el.tagName !== 'HTML' )
```
Comment (AvraamMavridis) [inline src/isElement.js]: That will fail in certain browsers. e.g. in Firefox the tag name of the `<html></html>` tag is `html`, not `HTML`

**Comment by nkashyap** [comment]
I tried to use it on couple of websites, e.g Amazon, etc and it basically crashed my browser (Chrome). Lot of libraries add css class on html tag elements to detect browser css capabilities which caused browser to crash

**Comment by AvraamMavridis** [comment]
I see. Ok, I will have a look on that. Today/Tomorrow and let you know. We will have to be sure that this solution works in the major browsers though.

**Comment by vwochnik** [comment]
I ran into the same issue. The problem, however, is not limited to the `<html>` element. Any element with a high number of classes will crash because `getCombinations()` is not limited.

Therefore, the only way to fix the problem is limiting `getCombinations()` to only chain a certain number of classes.

**Comment by AvraamMavridis** [comment]
@vwochnik  That can be a config option.

**Pull request closed without merging**